concepts/variables/introduction.md

```diff
@@ -1,6 +1,9 @@
 # Introduction
 
-In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package name is preferred to be the same as the folder it is located in.
+In Go an application is organized in packages.
+A package is a collection of source files located in the same folder.
+All source files in a folder must have the same package name at the top of the file.
+The package name is preferred to be the same as the folder it is located in.
```

Comment:
Why does the concept `variables` and `functions` both contain this section on packages? Shouldn't that be removed now that there is a package concept?

Reply:
This section can indeed be removed from this document.